scripts/release-redhat-certified.sh

```diff
@@ -2,6 +2,18 @@
 
 set -eou pipefail
 
+echo -n "Determining SHA for arm64 ... "
```

Comment:
did we already test these during 2.6.1 release? 
otherwise, how can we test this before the next release reliably?

Reply:
yes, I used this script to execute the PR